transport/rif_comms/utils.py

```diff
@@ -22,3 +30,8 @@ def notification_to_payload(notification_data: ChannelNewData) -> str:
         content = json.loads(content_data.decode())  # deserialize `data` dict
         return bytes(content["data"]).decode()  # deserialize `data.data` field
     return ""
+
+
+def get_sender_from_notification(notification_data: ChannelNewData) -> Address:
+    return to_canonical_address(notification_data.channelNewData.sender.address)
```

Comment:
```suggestion
def get_sender_from_notification(notification_data: ChannelNewData) -> Address:
    try:
        return to_canonical_address(notification_data.channelNewData.sender.address)
    except AttributeError:
        return Address(b'')
```
also, the type hint of the return should come from `eth_typing`, not `raiden.utils`